README.md

```diff
@@ -1,29 +1,29 @@
-# Sample Application with Scala and Scalatra #
+# Sample Application with Scala and Scalatra
 
-## Build & Run ##
+This sample is running on: https://scala-scalatra.is-easy-on-scalingo.com
 
-```sh
-$ cd sample-scala-scalatra
-$ ./sbt
-> jetty:start
-> browse
-```
-
-If `browse` doesn't launch your browser, manually open [http://localhost:8080/](http://localhost:8080/) in your browser.
-
-## Deploying on Scalingo
-
-You can use our one-click button: [![Deploy to Scalingo](https://cdn.scalingo.com/deploy/button.svg)](https://my.scalingo.com/deploy)
+## Deploy via Git
```

Comment:
Nitpick: as you wish
```suggestion
## Deploy via Git on Scalingo
```

Reply:
I don't think I'll change it because it's the template we have almost everywhere now.
For example: https://github.com/Scalingo/sample-node-nuxt